src/Parser/RetributionPaladin/Modules/Features/CastEfficiency.js

```diff
@@ -43,7 +43,7 @@ class CastEfficiency extends CoreCastEfficiency {
       spell: SPELLS.AVENGING_WRATH,
       category: CastEfficiency.SPELL_CATEGORIES.COOLDOWNS,
       getCooldown: haste => 120,
-      hideWithZeroCasts: true,
+      isActive: combatant => !combatant.hasTalent(SPELLS.CRUSADE_TALENT.id),
       recommendedCastEfficiency: 0.9,
     },
```

Comment:
Does Crusade really make it so that we're not interested in seeing the cast efficiency on AW?

Reply:
Crusade replaces AW

Reply:
my google foo failed me then 😓